course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -1059,6 +1089,22 @@ def test_data(self):
         serializer = ContainedCourseRunsSerializer(instance)
         self.assertDictEqual(serializer.data, instance)
 
+    def test_filter_contained_course_runs_on_edx_org_short_name(self):
+        """
+        Verify contained courses filtering on edX organization.
```

Comment:
Update method name and docstring, these are not understandable currently.
@iam-mhaseeb  Its ok if we have to spend 5 minutes for a good name, docstring which is compact, easy to read and easy to understand.

Reply:
Done